demo/fileExchange.sh

```diff
@@ -177,4 +177,5 @@ curl -X POST "http://$CONSUMER_IP:$CONSUMER_PORT/api/v1/data/transferprocess" \
 }
 EOF
 
+# Maybe echo something like: Transfer completed from <source> to <dest>
```

Comment:
The API call returns immediatly after the transfer has been requested. It's not clear at that point whether it actually worked. A full solution might be to repeatedly request the transfer status in a loop until it is either COMPLETED or FAILED. I just had a look at the logs of the EDC to see this, but could be a nifty addtion. I'm hesitating because the API is so poorly documented.